Cargo.toml

```diff
@@ -24,6 +24,11 @@ exclude       = [
 publish = false
 
 [features]
+os-poll = []
```

Comment:
Do what different features for epoll/kqueue/iocp? In the future we might add other selector, e.g. io_uring on Linux, do we provide any guarantee what selector is used?